plugins/de.cau.cs.kieler.spviz.spviz/src/de/cau/cs/kieler/spviz/spviz/generator/GenerateSyntheses.xtend

```diff
@@ -0,0 +1,1225 @@
+package de.cau.cs.kieler.spviz.spviz.generator
+
+import org.eclipse.xtext.generator.IFileSystemAccess2
+import java.util.ArrayList
+import de.cau.cs.kieler.spviz.spviz.sPViz.SPViz
+import de.cau.cs.kieler.spviz.spviz.sPViz.View
+import org.eclipse.emf.common.util.EList
+import java.util.LinkedHashMap
+import java.util.regex.Pattern
+import java.util.regex.Matcher
+
+/**
+ * Generates classes for the syntheses of the visualisation.
+ */
+class GenerateSyntheses {
+	def static void generate(IFileSystemAccess2 fsa, DataAccess spviz){
+		val folder = "viz/" + spviz.packageName.replace('.','/') + "/viz/"
+		
+		fsa.generateFile(folder + "DiagramSynthesis.xtend", generateDiagramSynthesis(spviz))
+		fsa.generateFile(folder + "KlighdSetup.xtend", generateKlighdSetup(spviz))
+		fsa.generateFile(folder + "Styles.xtend", generateStyles(spviz))
+		fsa.generateFile(folder + "SynthesisUtils.xtend", generateSynthesisUtils(spviz))
+		fsa.generateFile(folder + "SynthesisProperties.xtend", generateSynthesisProperties(spviz))
+//		fsa.generateFile(folder + "Options.xtend", generateOptions(vis.package))
+	}
+	
+	/**
+	 * Generates the contend for the diagram synthesis.
+	 * 
+	 * @param spviz
+	 * 		a DataAccess to easily get the information from
+	 * @return 
+	 * 		the generated file content as a string
+	 */
+	def static String generateDiagramSynthesis(DataAccess spviz){
```

Comment:
Style: Always have whitespace before the function's opening `{`. (Here and in all other cases.